etc/db-config.yaml

```diff
@@ -255,11 +255,16 @@
             default_value: false
             public: true
             description: Show results of TOO-LATE submissions in team interface?
-        -   name: show_balloons_postfreeze
+        -   name: minimum_number_of_balloons
+            type: int
+            default_value: 0
+            public: true
+            description: How many balloons to hand out ignoring freeze time. Only hands out balloons for problems solved pre-freeze.
+        -   name: any_balloon_postfreeze
```

Comment:
How is this option different than #Balloons>#Problems?

If the intent is to disclose all the information it's currently unclear.